### DISCUSSION TITLE
Should we release 0.69.11?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#68] [Patches]
## Released, please refer to #72

Current Release: [0.69.10](https://github.com/facebook/react-native/releases/tag/v0.69.10)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/facebook/react-native/pull/37452
2. https://github.com/facebook/react-native/commit/10d55888cce0b625c4b05a19ed609154893d17d0

#### Local commits to backport to main

1.

**Comment by gkasireddy202** [comment]
Thanks for your update.
Can I do any changes in RN0.69 or automatically fix this issue?

**Comment by kelset** [comment]
https://github.com/facebook/react-native/pull/37452

**Comment by cipolleschi** [comment]
We will need this for Xcode15 
- https://github.com/facebook/react-native/pull/37758

**Comment by robhogan** [comment]
Nominate this one for the RN-side fix to https://github.com/facebook/react-native/issues/36794 (to accompany a bump to Metro 0.70.4).
 - https://github.com/facebook/react-native/pull/37501 (https://github.com/facebook/react-native/commit/671ea383fe45dd9834a0c0481360de050df7f0c9)

**Reply by kelset** [reply]
we're in the process of releasing 0.69.11 - it will need to be done in 0.69.12 once the right bump of Metro/CLI will be out.

**Reply by robhogan** [reply]
Back in 0.69, Metro versions weren’t pinned - so for 0.69 only I think there’s value in picking this even before/without an explicit CLI bump - folks can get the benefit of the fix by reinstalling `node_modules`.

(No rush, of course - next time's fine)

**Reply by kelset** [reply]
we were already in the process of testing 0.69 branch when you posted this; so picking your commit and restarting would have slowed us down by quite a bit and we really wanted to push the 70 and 69 patches out today (we don't want to do releases on Fridays). But I'll add a note for next patch for sure.
